### DISCUSSION TITLE
Unsupported cipher or incorrect key length. Supported ciphers are: aes-128-cbc, aes-256-cbc, aes-128-gcm, aes-256-gcm.

### DISCUSSION THREAD

**Discussion by mrpmohiburrahman** [monicahq/monica#6450] [Q&A]
Hi, everyone,
I've deployed monicahq to heroku using "deploy to heroku" button.
APP_KEY, APP_URL, MAIL_FROM_ADDRESS, MAIL_NAME just as mentioned as in this page [https://github.com/monicahq/monica/blob/main/docs/installation/providers/heroku.md](https://github.com/monicahq/monica/blob/main/docs/installation/providers/heroku.md).

After deploying it, when I open the app it shows me the following error.

>  Whoops! Something went wrong.
>
> Unsupported cipher or incorrect key length. Supported ciphers are: aes-128-cbc, aes-256-cbc, aes-128-gcm, aes-256-gcm.

**Screenshots**
![Unsupported cipher](https://user-images.githubusercontent.com/35300157/204173699-4d3a1040-3705-44ce-ac83-dc16fea8bf1b.png)

**Additional context**
I am very new to this kind of stuff.
And never work with laravel.

Please, suggest me, how to solve this.

**Comment by Bag72** [comment]
Did anyone provide a solution to this?  I've hit exactly the same issue.

**Reply by Bag72** [reply]
Found it in the other section and it solved my issue.  This one should be closed as it is a duplicate.

https://github.com/monicahq/monica/issues/6449
